### PR TITLE
Add documentVersion doc

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: content/enterprise/reference-docs/server-api/document-version.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  // metadata source object (used by field extractor), e.g.
+  //   {
+  //     title: 'manual',
+  //   }
+  metadataSource,
+
+  // systemdata of a document, e.g.
+  // {
+  //   projectId: 3,
```
Comment (marcbachmann) [inline content/enterprise/reference-docs/server-api/document-version.md:72]: We currently don't have those camelCase values exposed:
https://github.com/livingdocsIO/livingdocs-server/blob/master/app/features/documents/document-version/document-version-model.js#L136-L143

Comment (peyerluk) [reply content/enterprise/reference-docs/server-api/document-version.md:72]: We could mention `publicSystemdata` for the camelCase variant.

File: content/enterprise/reference-docs/server-api/hooks.md [hunk 2]
```diff
@@ -122,35 +122,6 @@ async unpublishHookAsync ({documentType, payload}) {...}
 ```
 
 
-#### Info: The documentVersion object
```
Comment (peyerluk) [inline content/enterprise/reference-docs/server-api/hooks.md:125]: Isn't there a place in this file to add a link to the `documentVersion`?

File: content/enterprise/reference-docs/server-api/document-version.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+menus:
+  reference-docs:
+    parent: Server API
+---
+
+## The documentVersion object
+
+The `documentVersion` is a core object and is returned in a lot of server API calls. It contains information about a document like content, systemdata, metadata and more.
+
+## The documentVersion API
```
Comment (peyerluk) [inline content/enterprise/reference-docs/server-api/document-version.md:13]: 'The documentVersion API'
-> 'The documentVersion Type'
or
-> 'The documentVersion Interface'

File: content/enterprise/reference-docs/server-api/hooks.md (hunk 2 above)
Comment (DaRaFF) [reply content/enterprise/reference-docs/server-api/hooks.md:125]: I was also thinking about that. But then we should consequently add references to documentVersion in the whole documentation.

File: content/enterprise/reference-docs/server-api/document-version.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+title: DocumentVersion
+description: Model of a document on the server.
+menus:
+  reference-docs:
+    parent: Server API
+---
+
+## The documentVersion object
+
+The `documentVersion` is a core object and is returned in a lot of server API calls. It contains information about a document like content, systemdata, metadata and more.
```
Comment (peyerluk) [inline content/enterprise/reference-docs/server-api/document-version.md:11]: 'and more.' -> 'and the latest publication'

**Review by peyerluk — APPROVED** [review]
Looks good. Implement feedback as you see fit.

File: content/enterprise/reference-docs/server-api/hooks.md (hunk 2 above)
Comment (peyerluk) [reply content/enterprise/reference-docs/server-api/hooks.md:125]: Don't let a big improvement stay in the way of a small one :)

**Pull request merged**